app.py

```diff
@@ -43,6 +38,36 @@ def group_category(group=[], category='all'):
         result["count"] = len(posts)
         result["query"] = search
         return flask.render_template('search.html', result=result)
+
+    page = flask.request.args.get('page')
+    posts, metadata = api.get_posts(page=page, per_page=13)
+
+    webinars = get_rss_feed_content(
+        'https://www.brighttalk.com/channel/6793/feed/rss'
+    )
+
+    featured_post = api.get_featured_post()
+    homepage_posts = []
+
+    for post in posts:
+        if post['id'] != featured_post['id']:
+            homepage_posts.append(post)
```

Comment:
If the featured_post is not in the first 13 posts then homepage_posts will be 13 posts long. Which will result in an orphan item in the template. Would be worth checking if the count has reached 12 and breaking.

Reply:
Good catch